DigitalExcellence/dex-frontend
# Feature/highlight a project

## Description

Added a modal for the highlight a project to set a start and end date.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Login to DeX
2. Go to a project detail page
3. Click on the "Highlight project" 
4. fill in a start and end date (or check the never ending)
5. project should be added to the database and randomly show up as an highlighted project.

## Link to issue

Closes: #75
